deployment/E_heroku.md

```diff
@@ -163,7 +149,7 @@ config :logger, level: :info
 config :hello_phoenix, HelloPhoenix.Repo,
   adapter: Ecto.Adapters.Postgres,
   url: System.get_env("DATABASE_URL"),
-  pool_size: 20,
+  pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10"),
```

Comment:
I believe this variable will need to be available at build time so setting it here https://github.com/phoenixframework/phoenix_guides/pull/514/files#diff-969f42064dcef84e92928f7483e56814R220 will be required.